parkings/forecasts/cache_parking_counts.py

```diff
@@ -0,0 +1,73 @@
+import datetime
+
+from parkings.models import Parking, ParkingArea, ParkingCount, Region
+
+
+def cache_parking_counts_by_region():
+    parkings_start_instance = (
+        ParkingCount.objects.filter(region__isnull=False, is_forecast=False)
+        .order_by("time")
+        .last()
+    )
+    if parkings_start_instance:
+        parkings_start = parkings_start_instance.time
+    else:
+        parkings_start = Parking.objects.all().order_by("time_start").first().time_start
+
+    parkings_end = Parking.objects.all().order_by("-time_start").first().time_end
```

Comment:
Sorry, didn't notice this earlier, but there's still a bug in this line, since time_end could be null.

Maybe it should rather use `end_time = timezone.now()`, since it  might be that the parkings end before current time, but we would still want to generate the 0 counts for those hours between the last parking and current time.